src/Dao/PlagiabotDao.php

```diff
@@ -38,20 +38,19 @@ class PlagiabotDao extends AbstractDao {
 	 *     'ithenticate_id'
 	 *   integer 'id' exact ithenticate_id of a record. This will override all
 	 *   	 other filter options
-	 *   string 'wikiprojects' pipe-separated list of wikiprojects
 	 *   string 'wikiLang' The language code of the Wikipedia to query for
```

Comment:
`searchText` and `searchCriteria` should be added to this list. Also note the other options are underscored, e.g. `last_id` vs. `lastId`, `filter_user` vs. `filterUser`. Doesn't really matter, but might be good to be consistent.